policies/TrademarkPolicy.md

```diff
@@ -0,0 +1,215 @@
+# Trademark Policy
+
+#### Last modified 2022-11-22
+
+
+
+# Purpose
+
+OpenSSL is committed to promoting the use of its open source software. While 
+open source software is generally free to download and modify, the use of
+open source software does not include the right to OpenSSL trademarks.
+
+The Trademark Policy (The Policy) aims to protect and ensure consistent usage 
+of the OpenSSL trademarks and to clarify how the OpenSSL trademarks may be 
+used.
+
+As a part of this process, the OpenSSL trademark is a registered United States 
+trademark of the OpenSSL Software Foundation. 
+For registration outside the United States, we have filed a [Madrid Protocol](http://en.wikipedia.org/wiki/Madrid_system) 
+application to extend the protection in the European Union, China and Japan.
```

Comment:
and other locations.

Reply:
@mattcaswell Do we need to list the specific locations (do we know what these are?)